src/components/Photo/style.scss

```diff
@@ -67,13 +67,13 @@
 }
 
 .selfieIcon {
-  background-image: url('./assets/selfie-picto.svg');
   background-size: contain;
+  background-image: url('./assets/selfie-picto.svg');
 }
 
 .glassesIcon {
-  background-image: url('./assets/glasses-picto.svg');
   background-size: contain;
+  background-image: url('./assets/glasses-picto.svg');
```

Comment:
Oh it's the same issue again...

Reply:
yeap. but this time it seems to only happen on iPhone 12. 😞 
I think my attempt at reducing code duplication ended up causing this weird Safari bug, so I might just avoid using the Sass `@extend` here while I work out why exactly does this screen seems to suffer from this problem.